src/main/java/teammates/ui/automated/AutomatedActionFactory.java

```diff
@@ -18,19 +20,23 @@
             new HashMap<String, Class<? extends AutomatedAction>>();
     
     static {
-        map("/auto/adminEmailPrepareTaskQueueWorker", null);
-        map("/auto/adminEmailWorker", null);
+        // Cron jobs
         map(ActionURIs.AUTOMATED_LOG_COMPILATION, CompileLogsAction.class);
-        map("/auto/courseJoinRemindEmailWorker", null);
-        map("/auto/emailWorker", null);
-        map("/auto/feedbackRemindEmailParticularUsersWorker", null);
-        map("/auto/feedbackRemindEmailWorker", null);
         map(ActionURIs.AUTOMATED_FEEDBACK_OPENING_REMINDERS, FeedbackSessionOpeningRemindersAction.class);
         map(ActionURIs.AUTOMATED_FEEDBACK_CLOSED_REMINDERS, FeedbackSessionClosedRemindersAction.class);
         map(ActionURIs.AUTOMATED_FEEDBACK_CLOSING_REMINDERS, FeedbackSessionClosingRemindersAction.class);
         map(ActionURIs.AUTOMATED_FEEDBACK_PUBLISHED_REMINDERS, FeedbackSessionPublishedRemindersAction.class);
+        
+        // Task queue workers
+        map(TaskQueue.ADMIN_PREPARE_EMAIL_WORKER_URL, AdminPrepareEmailWorkerAction.class);
+        map(TaskQueue.ADMIN_SEND_EMAIL_WORKER_URL, AdminSendEmailWorkerAction.class);
+        map(TaskQueue.COURSE_JOIN_REMIND_EMAIL_WORKER_URL, CourseJoinRemindEmailWorkerAction.class);
+        map("/auto/emailWorker", null);
```

Comment:
Just to confirm, will `"/auto/emailWorker"` and `"/auto/feedbackSubmissionAdjustmentWorker"` map to non-null values in a future PR?